examples/app-basic/tests/appConfig.spec.ts

```diff
@@ -0,0 +1,22 @@
+import { test, expect } from './fixtures';
+
+test('configuring app', async ({ appConfigPage, page }) => {
+  await appConfigPage.goto();
+
+  const saveForm = page.getByRole('button', { name: /Save API settings/i });
+  await expect(saveForm).toBeVisible();
```

Comment:
Unlike in cypress, you generally don't need to check visibility. All Playwright actions such as the click method below performs [actionability](https://playwright.dev/docs/actionability#introduction) checks that will do retries until it succeeds or fails.